tests/testsuite/support/mod.rs

```diff
@@ -202,6 +204,9 @@ impl SymlinkBuilder {
 
 pub struct Project {
     root: PathBuf,
+    start: FileTime,
+    updates: HashMap<PathBuf, FileTime>,
+    next_update: i64,
```

Comment:
Do you think it's worth documenting these?

Reply:
Oops yes, meant to do that already!